classes/core/PKPApplication.inc.php

```diff
@@ -903,10 +903,15 @@ public static function getIntMaxFileMBs()
         $scale = strtolower(substr(UPLOAD_MAX_FILESIZE, -1));
         switch ($scale) {
             case 'g':
-                $num = $num / 1024;
-                // no break
-            case 'k':
                 $num = $num * 1024;
+                break;
+            case 'k':
+                $num = $num / 1024;
+                break;
+            default:
+                // No suffix or nothing matched, so this is "b" (Byte)
+                $num = $num / 1024 / 1024
```

Comment:
I think this will shear off the last digit of the byte count, no?

Reply:
(Also missing `;`)

Reply:
It will shear off the last digit? What do you mean?

Ill fix the typo asap - dint ran the code, just edited it here.

Reply:
Ah, that shearing off... Got it!